README.md

```diff
@@ -14,7 +14,7 @@ I do not support piracy in any way, all the data and information in my software
 ## Current functionality
 ### General
 - Support for Web Browser (SPIDER) entrypoint for Old3DS (ver 4.1-9.2)
-- Support for System Settings (MSET) entrypoint for Old3DS (ver 4.1-4.5 and 6.0-6.4)
+- Support for System Settings (MSET) entrypoint for Old3DS (6.0-6.4)
```

Comment:
I plan to continue using 9.2 with a downgraded MSET on my o3DS because it boots much faster than 4.5 does, so if there's a way I can figure out what combinations work I'd be more than happy to test and document them all.